release/datafiles/colormanagement/adaptation.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import PyOpenColorIO
```

Comment:
This stuff is Troy's magic which I don't fully understand, but it is responsible for the display of the wide-gamut data which might come out of the spectral engine. If you just clip spectral colours down into a colour space like sRGB, you end up with some pretty awful results. I think this is still a work in progress, as he tries to create a new 'filmic' which can render the entire spectral locus, so there's some serious gamut mapping to be done there.